src/shared/contexts/query.tsx

```diff
@@ -588,9 +589,14 @@ export const QueryProvider: FC = ({children}) => {
           let didTruncate = false
           let read = await reader.read()
 
-          const BYTE_LIMIT =
+          let BYTE_LIMIT =
             getFlagValue('increaseCsvLimit') ?? FLUX_RESPONSE_BYTES_LIMIT
 
+          if (!window.location.pathname.includes(PROJECT_NAME.toLowerCase())) {
+            BYTE_LIMIT =
+              getFlagValue('dataExplorerCsvLimit') ?? FLUX_RESPONSE_BYTES_LIMIT
+          }
```

Comment:
Ah...doesn't this add the new feature flag `'dataExplorerCsvLimit'` to only the notebooks? And not the new Qx Builder?